acer-wmi-battery.c

```diff
@@ -217,6 +253,28 @@ static void update_state(void)
 			battery_status.health_mode ? "enabled" : "disabled");
 }
 
+static ssize_t temperature_show(struct device_driver *driver, char *buf)
+{
+	acpi_status status;
+	u32 value;
+
+	/*
+	 * We always use battery number 0x1 to stay consistent with the
```

Comment:
This comment could be a bit misleading. There don't seem to be any relevant Acer laptops with multiple batteries. At least, the Acer Care Center does always use this constant battery index, too. We could define a "battery_index" constant at the top of the source file and add an explanation ("This driver supports only laptops with a single battery") there. What do you think?

Reply:
Good idea, which version of Acer Care Center only uses battery index 0x1?

Reply:
The one for the Acer Swift 3 (SF314-34) for sure, but I decompiled two or three different versions a long time ago, I don't remember which ones unfortunately.

Reply:
Ok.